src/software/ai/hl/stp/play/ball_placement/ball_placement_play_fsm.h

```diff
@@ -3,21 +3,25 @@
 #include "proto/parameters.pb.h"
 #include "shared/constants.h"
 #include "software/ai/hl/stp/play/play_fsm.h"
-#include "software/ai/hl/stp/tactic/dribble/dribble_tactic.h"
-#include "software/ai/hl/stp/tactic/move/move_tactic.h"
-#include "software/ai/hl/stp/tactic/pivot_kick/pivot_kick_tactic.h"
+#include "software/ai/hl/stp/skill/move/move_skill.h"
+#include "software/ai/hl/stp/tactic/assigned_skill/specialized_assigned_skill_tactics.h"
 #include "software/ai/passing/eighteen_zone_pitch_division.h"
-
+#include "software/geom/algorithms/closest_point.h"
 
 using Zones = std::unordered_set<EighteenZoneId>;
 
 struct BallPlacementPlayFSM
 {
+    double BACK_AWAY_FROM_CORNER_EXTRA_M               = 0.9;
+    double BACK_AWAY_FROM_WALL_M                       = ROBOT_MAX_RADIUS_METERS * 5.5;
+    double MINIMUM_DISTANCE_FROM_WALL_FOR_ALIGN_METERS = ROBOT_MAX_RADIUS_METERS * 4.0;
```

Comment:
nit: some of these comments have `_M` suffix and others have a `_METERS` suffix. We should make this consistent across all the constants here. I prefer the `_M` prefix.